apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/codegen/kotlin/KotlinSymbols.kt

```diff
@@ -67,6 +66,20 @@ internal object KotlinClassNames {
   val List = ClassName("kotlin.collections", "List")
   val Map = ClassName("kotlin.collections", "Map")
   val Array = ClassName("kotlin", "Array")
+
+  /**
+   * Adapters
+   */
+  val AnyAdapter = MemberName(ClassNames.apolloApiPackageName, "AnyAdapter")
+  val BooleanAdapter = MemberName(ClassNames.apolloApiPackageName, "BooleanAdapter")
+  val DoubleAdapter = MemberName(ClassNames.apolloApiPackageName, "DoubleAdapter")
+  val IntAdapter = MemberName(ClassNames.apolloApiPackageName, "IntAdapter")
+  val StringAdapter = MemberName(ClassNames.apolloApiPackageName, "StringAdapter")
+  val NullableAnyAdapter = MemberName(ClassNames.apolloApiPackageName, "NullableAnyAdapter")
+  val NullableBooleanAdapter = MemberName(ClassNames.apolloApiPackageName, "NullableBooleanAdapter")
+  val NullableDoubleAdapter = MemberName(ClassNames.apolloApiPackageName, "NullableDoubleAdapter")
+  val NullableIntAdapter = MemberName(ClassNames.apolloApiPackageName, "NullableIntAdapter")
+  val NullableStringAdapter = MemberName(ClassNames.apolloApiPackageName, "NullableStringAdapter")
```

Comment:
I had to renamed `KotlinClassNames` to `KotlinSymbols` because I put MemberNames here. We could very well ad them elsewhere to keep the `KotlinClassNames` name